From 67475a2d864cc2f63cca2b9bbe2b357db027ef6b Mon Sep 17 00:00:00 2001
From: Johan Hovold <jhovold@gmail.com>
Date: Sun, 29 Dec 2013 19:22:59 +0100
Subject: [PATCH 1/5] USB: pl2303: remove bogus delta_msr_wait wake up
Git-Commit: 71c671bf4cead0e801eccbed3fe790e817f6b6d4
Patch-Mainline: v3.14
References: bnc#959649

Remove bogus MSR wait-queue wake up from process_read_urb which never
updates the MSR flags.

Signed-off-by: Johan Hovold <jhovold@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Conflicts:
	drivers/usb/serial/pl2303.c
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/serial/pl2303.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 5532ea5..97d7dd1 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -792,7 +792,6 @@ static void pl2303_process_read_urb(struct urb *urb)
 	line_status = priv->line_status;
 	priv->line_status &= ~UART_STATE_TRANSIENT_MASK;
 	spin_unlock_irqrestore(&priv->lock, flags);
-	wake_up_interruptible(&priv->delta_msr_wait);
 
 	if (!urb->actual_length)
 		return;
-- 
2.1.4

